package.json

```diff
@@ -32,6 +32,7 @@
     "test-ci": "cross-env NODE_ENV=test npm run lint && npm run flow && npm run test"
   },
   "dependencies": {
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
```

Comment:
This is in `devDependencies` already, please remove from `dependencies`. This field is only meant for runtime dependencies of this package.